### PR TITLE
TETH lore from vanilla LC + Penitent Girl

### PR DESCRIPTION
**Pull request by RedMageCole**
## About The Pull Request

This late-night PR that I finished on a whim adds lore from TETH-class Abnormalities! This set particularly includes all of the Lobotomy Corporation vanilla Abnormalities. To continue with my goal to upload custom lore with every set, I've included additional custom lore both for Penitent Girl as well as some additional lore for Standard Training Dummy Rabbit based on artbook knowledge!

## Why It's Good For The Game

Reading is scientifically proven to make readers somewhere between 0% and 100% less likely to chop off their feet. Don't take the chance.

## Changelog
:cl:
add: Added Standard Training Dummy Rabbit lore
add: Added Spider Bud lore
add: Added Punishing Bird lore
add: Added Fragment of the Universe lore
add: Added Beauty and the Beast lore
add: Added 1.76 MHz lore
add: Added Crumbling Armor lore
add: Added Bloodbath lore
add: Added Old Lady lore
add: Added Forsaken Murderer lore
add: Added Ppodae lore
add: Added Today's Shy Look lore
add: Added Grave of Cherry Blossoms lore
add: Added Penitent Girl lore
add: Added Void Dream lore
add: Added Meat Lantern lore
add: Added The Lady Facing the Wall lore
/:cl:

### CODE REVIEW

File: code/modules/paperwork/records/lore/teth.dm [hunk 1]
```diff
@@ -1,21 +1,438 @@
...
+	Old Lady	<br>
+	"An Abnormality taking the form of an old and feeble woman. Anytime someone enters its Containment Unit, it will talk to that person."<br>
+
+	"What it tells differs from time to time, however, it will always commence with �Dearie, dearie, would you like to hear a story?�. This story could be a lullaby, an unspecified poem, a nursery rhyme, a common children�s story, or something entirely new.<br>
+	It knows every children�s story and will not stop recounting until the listener finally leaves."<br>
+
+	<h3><center>Excerpt from Counseling Log</center></h3><br>
+	She may be the loneliest of all the Abnormalities. She�s filled with loneliness, and tells her tales to fill that void. Well, every story needs a listener. When someone enters her room, she�ll try to fill that void. Of course, listening to her is a very dangerous thing. She knows every story on Earth and even those that cannot possibly exist. Our minds can�t cope with such a thing. She could fry your brain in an instant.}
+
+//Forsaken Murderer
```
Comment (Kitsunemitsu) [inline code/modules/paperwork/records/lore/teth.dm:213]: Forgot  " at the end of the last lore file

**Review by Kitsunemitsu — CHANGES_REQUESTED** [review]
Minor format issues

File: code/modules/paperwork/records/lore/teth.dm (hunk 1 above)
Comment (RedMageCole) [reply code/modules/paperwork/records/lore/teth.dm:213]: Good catch, thank you! Just fixed it.

**Review by Kitsunemitsu — APPROVED** [review]
Issues fixed, looks good to me

**Pull request merged**